static/riotjs/wizard/participants_table.tag

```diff
@@ -365,4 +368,10 @@
                 })
         }
     </script>
+    <style>
+        div.right_and_left {
```

Comment:
Rename `right_and_left` to `space-between`.  Generally, we use hyphens rather than underscores for html attribute values, including CSS class names.

Reply:
I see... Modified